packages/devtools_app/lib/src/service/editor/api_classes.dart

```diff
@@ -0,0 +1,418 @@
+// Copyright 2024 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// Known kinds of events that may come from the editor.
+///
+/// This list is not guaranteed to match actual events from any given editor as
+/// the editor might not implement all functionality or may be a future version
+/// running against an older version of this code/DevTools.
+enum EditorEventKind {
+  // Device.
+  deviceAdded,
+  deviceRemoved,
+  deviceChanged,
+  deviceSelected,
+
+  // Debug Session.
+  debugSessionStarted,
+  debugSessionChanged,
+  debugSessionStopped,
```

Comment:
can you add more dart doc here on what these events are for and also what their payload will look like if they come with arguments that are expected to be parsed?

Reply:
I've added a short comment to each with a reference to the class that the event goes with (which is the payload for such an event). Is this enough or do you think it needs more?